components/style/stylesheets.rs

```diff
@@ -112,7 +120,7 @@ pub struct KeyframesRule {
 #[derive(Debug)]
 pub struct MediaRule {
     pub media_queries: Arc<RwLock<MediaList>>,
-    pub rules: Vec<CSSRule>,
+    pub rules: CSSRules,
```

Comment:
I see, thanks. I think this is ok for now, and I expect that once we have CSSOM functional in Servo and Stylo we’ll likely to a pass at simplifying/optimizing things.